Impossible to install both Valkyrie plugin and "Compose Multiplatform IDE support" plugin

First, thanks for your amazing plugin! It's very useful in my workflow.

In Android Studio, I use the [Compose Multiplatform IDE support plugin](https://plugins.jetbrains.com/plugin/16541-compose-multiplatform-ide-support).
 
When I install Valkyrie, after restarting Android Studio, it automatically uninstalls Compose Multiplatform IDE support plugin.
Also, if I tried to reinstall Compose Multiplatform IDE support plugin, after restarting Android Studio, it uninstalls Valkyrie.

Do you have any idea why this happens? And is there a workaround for this issue? Thanks for your help!

- Tested on a fresh installation of Android Studio
- Android Studio Ladybug Feature Drop | 2024.2.2

https://github.com/user-attachments/assets/bd00e459-701f-4ca2-a5b8-daa54d07c50a

Today I retried in Android Studio Canary 5 and I got this stacktrace:

```
com.intellij.diagnostic.PluginException: Cannot create extension (class=io.github.composegears.valkyrie.editor.ImageVectorPreviewEditorProvider) [Plugin: io.github.composegears.valkyrie]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:989)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:73)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:33)
	at com.intellij.openapi.extensions.ExtensionPointName.findByIdOrFromInstance(ExtensionPointName.kt:250)
	at com.intellij.openapi.fileEditor.impl.FileEditorProviderManagerImpl.getProvider(FileEditorProviderManagerImpl.kt:188)
	at com.intellij.openapi.fileEditor.impl.HistoryEntryKt.parseEntry(HistoryEntry.kt:148)
	at com.intellij.openapi.fileEditor.impl.HistoryEntryKt.access$parseEntry(HistoryEntry.kt:1)
	at com.intellij.openapi.fileEditor.impl.HistoryEntry$Companion.createHeavy(HistoryEntry.kt:46)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager.loadState(EditorHistoryManager.kt:319)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager.loadState(EditorHistoryManager.kt:30)
	at com.intellij.configurationStore.ComponentStoreImpl.doInitComponent(ComponentStoreImpl.kt:510)
	at com.intellij.configurationStore.ComponentStoreImpl.initComponent(ComponentStoreImpl.kt:430)
	at com.intellij.configurationStore.ComponentStoreImpl.initComponent(ComponentStoreImpl.kt:145)
	at com.intellij.configurationStore.ComponentStoreWithExtraComponents.initComponent(ComponentStoreWithExtraComponents.kt:46)
	at com.intellij.serviceContainer.ComponentManagerImpl.initializeService$intellij_platform_serviceContainer(ComponentManagerImpl.kt:657)
	at com.intellij.serviceContainer.LightServiceInstanceSupport$LightServiceInstanceInitializer.createInstance(LightServiceInstanceSupport.kt:56)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:163)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:161)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:27)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:90)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:123)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:52)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:146)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:14)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:136)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:96)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext$suspendImpl(LazyInstanceHolder.kt:88)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext(LazyInstanceHolder.kt)
	at com.intellij.serviceContainer.ComponentManagerImplKt$doGetOrCreateInstanceBlocking$1.invokeSuspend(ComponentManagerImpl.kt:1566)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:277)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:111)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$BuildersKt__BuildersKt(Builders.kt:84)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:52)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization$lambda$10(ComponentManagerImpl.kt:1684)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext(context.kt:81)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization(ComponentManagerImpl.kt:1675)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doGetOrCreateInstanceBlocking(ComponentManagerImpl.kt:1565)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1560)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:751)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:695)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager$Companion.getInstance(EditorHistoryManager.kt:411)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager$MyEditorManagerListener.service_delegate$lambda$0(EditorHistoryManager.kt:368)
	at kotlin.UnsafeLazyImpl.getValue(Lazy.kt:81)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager$MyEditorManagerListener.getService(EditorHistoryManager.kt:368)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager$MyEditorManagerListener.selectionChanged$lambda$3(EditorHistoryManager.kt:399)
	at com.intellij.psi.impl.PsiDocumentManagerBase.performWhenAllCommitted(PsiDocumentManagerBase.java:628)
	at com.intellij.psi.impl.PsiDocumentManagerBase.performWhenAllCommitted(PsiDocumentManagerBase.java:618)
	at com.intellij.openapi.fileEditor.impl.EditorHistoryManager$MyEditorManagerListener.selectionChanged(EditorHistoryManager.kt:381)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:768)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:712)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:451)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:430)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:493)
	at jdk.proxy2/jdk.proxy2.$Proxy185.selectionChanged(Unknown Source)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.fireSelectionChanged$lambda$58(FileEditorManagerImpl.kt:1797)
	at com.intellij.openapi.application.CoroutinesKt.writeIntentReadAction$lambda$1$lambda$0(coroutines.kt:329)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:917)
	at com.intellij.openapi.application.CoroutinesKt.writeIntentReadAction$lambda$1(coroutines.kt:329)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContextInner(coroutines.kt:341)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invokeSuspend(coroutines.kt:233)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:232)
	at com.intellij.openapi.application.CoroutinesKt.writeIntentReadAction(coroutines.kt:328)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.fireSelectionChanged(FileEditorManagerImpl.kt:1796)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.access$fireSelectionChanged(FileEditorManagerImpl.kt:118)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$2$1$1.invokeSuspend(FileEditorManagerImpl.kt:267)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at com.intellij.openapi.application.impl.DispatchedRunnable.run(DispatchedRunnable.kt:44)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:236)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:117)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:675)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:573)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:857)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16(IdeEventQueue.kt:354)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:1045)
	at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:917)
	at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
	at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:1045)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:395)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: io.github.composegears.valkyrie.editor.ImageVectorPreviewEditorProvider PluginClassLoader(plugin=PluginDescriptor(name=Valkyrie - SVG to ImageVector, id=io.github.composegears.valkyrie, descriptorPath=plugin.xml, path=~/Library/Application Support/Google/AndroidStudio2024.3.2/plugins/idea-plugin, version=0.11.1, package=null, isBundled=false), packagePrefix=null, state=active, parents=PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.features-trainer, descriptorPath=kotlin.features-trainer.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.training, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.project-wizard.maven, descriptorPath=kotlin.project-wizard.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.tools.projectWizard.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.project-wizard.gradle, descriptorPath=kotlin.project-wizard.gradle.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.tools.projectWizard.gradle, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.project-wizard.idea, descriptorPath=kotlin.project-wizard.idea.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.tools.projectWizard, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.project-wizard.compose, descriptorPath=kotlin.project-wizard.compose.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.tools.composeProjectWizard, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.onboarding.gradle, descriptorPath=kotlin.onboarding.gradle.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.onboarding.gradle, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.onboarding, descriptorPath=kotlin.onboarding.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.onboarding, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.j2k.k2, descriptorPath=kotlin.j2k.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.j2k.k2, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.run-configurations.testng, descriptorPath=kotlin.run-configurations.testng.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.testng, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.searching.k2, descriptorPath=kotlin.searching.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.searching, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-reference-index, descriptorPath=kotlin.compiler-reference-index.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.search.refIndex, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.run-configurations.jvm, descriptorPath=kotlin.run-configurations.jvm.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.runConfigurations.jvm, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.navigation, descriptorPath=kotlin.navigation.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.navigation, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.ml-completion, descriptorPath=kotlin.ml-completion.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.mlCompletion, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.compiler-plugin-support.maven, descriptorPath=kotlin.compiler-plugins.compiler-plugin-support.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.maven.compilerPlugin, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.maven, descriptorPath=kotlin.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.live-templates.k2, descriptorPath=kotlin.code-insight.live-templates.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.liveTemplates.k2, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.live-templates.shared, descriptorPath=kotlin.code-insight.live-templates.shared.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.liveTemplates, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.refactorings.rename.k2, descriptorPath=kotlin.refactorings.rename.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.refactoring.rename, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.refactorings.move.k2, descriptorPath=kotlin.refactorings.move.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.refactoring.move, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.refactorings.k2, descriptorPath=kotlin.refactorings.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.refactoring, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.injection.k2, descriptorPath=kotlin.injection.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.injection, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.jvm-debugger.evaluation.k2, descriptorPath=kotlin.jvm-debugger.evaluation.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.debugger, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.structural-search.k2, descriptorPath=kotlin.code-insight.structural-search.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.codeinsight.structuralsearch, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.intentions.k2, descriptorPath=kotlin.code-insight.intentions.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.codeinsight.intentions, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.inspections.k2, descriptorPath=kotlin.code-insight.inspections.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.codeinsight.inspections, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.fixes.k2, descriptorPath=kotlin.code-insight.fixes.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.codeinsight.fixes, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.k2, descriptorPath=kotlin.code-insight.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.k2.codeinsight, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.jvm-decompiler, descriptorPath=kotlin.jvm-decompiler.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.jvmDecompiler, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.run-configurations.junit, descriptorPath=kotlin.run-configurations.junit.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.junit, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.k2.internal, descriptorPath=kotlin.k2.internal.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.internal.actions, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.i18n, descriptorPath=kotlin.i18n.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.i18n, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.highlighting.k2, descriptorPath=kotlin.highlighting.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.highlighting, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.code-insight-groovy, descriptorPath=kotlin.gradle.code-insight-groovy.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.groovy, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.grazie, descriptorPath=kotlin.grazie.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.grazie, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.gradle-tooling, descriptorPath=kotlin.gradle.gradle-tooling.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.gradleTooling, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.gradle-java, descriptorPath=kotlin.gradle.gradle-java.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.gradleJava, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.code-insight-common, descriptorPath=kotlin.gradle.code-insight-common.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.gradleCodeInsightCommon, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.code-insight-toml, descriptorPath=kotlin.gradle.code-insight-toml.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.gradle.versionCatalog.toml, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.gradle.gradle, descriptorPath=kotlin.gradle.gradle.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.gradle, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.git, descriptorPath=kotlin.git.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.git, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.coverage, descriptorPath=kotlin.coverage.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.coverage, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.copyright, descriptorPath=kotlin.copyright.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.copyright, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.scripting, descriptorPath=kotlin.compiler-plugins.scripting.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.scripting.gradleJava, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.sam-with-receiver.maven, descriptorPath=kotlin.compiler-plugins.sam-with-receiver.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.samWithReceiver.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.noarg.maven, descriptorPath=kotlin.compiler-plugins.noarg.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.noarg.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.lombok.maven, descriptorPath=kotlin.compiler-plugins.lombok.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.lombok.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.kotlinx-serialization.maven, descriptorPath=kotlin.compiler-plugins.kotlinx-serialization.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.kotlinxSerialization.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.kotlinx-serialization.k2, descriptorPath=kotlin.compiler-plugins.kotlinx-serialization.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.kotlinxSerialization.k2, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.kapt, descriptorPath=kotlin.compiler-plugins.kapt.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.kapt.gradleJava, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.assignment.maven, descriptorPath=kotlin.compiler-plugins.assignment.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.assignment.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.assignment.common.k2, descriptorPath=kotlin.compiler-plugins.assignment.common.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.assignment, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.compiler-plugins.allopen.maven, descriptorPath=kotlin.compiler-plugins.allopen.maven.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.compilerPlugin.allopen.maven, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.postfix-templates.k2, descriptorPath=kotlin.code-insight.postfix-templates.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.codeInsight.postfix, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.line-markers.k2, descriptorPath=kotlin.code-insight.line-markers.k2.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.codeInsight.lineMarkers, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.intentions.shared, descriptorPath=kotlin.code-insight.intentions.shared.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.codeInsight.intentions.shared, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.code-insight.inspections.shared, descriptorPath=kotlin.code-insight.inspections.shared.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.codeInsight.inspections.shared, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.injection, descriptorPath=kotlin.base.injection.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.injection, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.fir.project-structure, descriptorPath=kotlin.base.fir.project-structure.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.fir.projectStructure, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.fir.code-insight, descriptorPath=kotlin.base.fir.code-insight.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.fir.codeInsight, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.fir.analysis-api-platform, descriptorPath=kotlin.base.fir.analysis-api-platform.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.fir.analysisApiPlatform, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.external-build-system, descriptorPath=kotlin.base.external-build-system.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.externalSystem, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.base.analysis-api-platform, descriptorPath=kotlin.base.analysis-api-platform.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin.idea.base.analysisApiPlatform, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, moduleName=kotlin.performanceExtendedPlugin, descriptorPath=kotlin.performanceExtendedPlugin.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=com.intellij.performance.performancePlugin, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.impl, descriptorPath=intellij.platform.vcs.impl.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/lib, version=243.23654.117, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.log.impl, descriptorPath=intellij.platform.vcs.log.impl.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/lib, version=243.23654.117, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.dvcs.impl, descriptorPath=intellij.platform.vcs.dvcs.impl.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/lib, version=243.23654.117, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.collaborationTools, descriptorPath=intellij.platform.collaborationTools.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/lib, version=243.23654.117, package=null, isBundled=true), PluginDescriptor(name=Kotlin, id=org.jetbrains.kotlin, descriptorPath=plugin.xml, path=~/Applications/Android Studio Meerkat Feature Drop 2024.3.2 Canary 5.app/Contents/plugins/Kotlin, version=243.23654.117.2432.13058383-AS, package=org.jetbrains.kotlin, isBundled=true), )
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1462)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass$default(ComponentManagerImpl.kt:1458)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:896)
	at com.intellij.openapi.extensions.impl.InterfaceExtensionImplementationClassResolver.resolveImplementationClass(InterfaceExtensionImplementationClassResolver.kt:21)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:51)
	... 118 more

```

--- a/idea-plugin/build.gradle.kts
+++ b/idea-plugin/build.gradle.kts
@@ -46,6 +46,7 @@ compose.resources {
 
 intellijPlatform {
     buildSearchableOptions = false
+    projectName = "valkyrie-plugin"
     pluginConfiguration.ideaVersion {
         sinceBuild = "241"
         untilBuild = provider { null }

